### ISSUE TITLE
Images drag and drop is broken

### ISSUE THREAD

**Issue by epikgit** [ryanmcalister/unotes#126]
Hi,
with the last VS Code update (v1.67.1) the images d&d doesn't work anymore.
To reproduce that, open a note and d&d an image on the wysiwyg panel. VS Code will open a different page to preview the image.

Thank you

**Comment by ryanmcalister** [comment]
Yes, thanks for reporting this. It looks like it does still work but you must hold the `shift` key down while dropping. This was a change in VS Code apparently.
